ui/app/src/Request/Alert.elm

```diff
@@ -0,0 +1,53 @@
+{-
+   Alertmanager API
+   API of the Prometheus Alertmanager (https://github.com/prometheus/alertmanager)
+
+   OpenAPI spec version: 0.0.1
+
+   NOTE: This file is auto generated by the openapi-generator.
+   https://github.com/openapitools/openapi-generator.git
+   Do not edit this file manually.
+-}
+
+
+module Request.Alert exposing (getAlerts, postAlerts)
+
+import Data.Alert exposing (Alert, alertDecoder, alertEncoder)
+import Dict
+import Http
+import Json.Decode as Decode
+
+
+basePath : String
+basePath =
+    "http://localhost"
```

Comment:
How does this interact with using ports (elm-reactor will want to communicate on a different port than it's being served on, for example)?